### DISCUSSION TITLE
如何使指定域名不经过核心

### DISCUSSION THREAD

**Discussion by Journalist-HK** [vernesong/OpenClash#2961] [Q&A]
由于路由器性能问题，经过核心但直连的流量跑不满带宽，设置里提供了国内ip绕过、本地ip绕过、国内ip域名强制走核心这些选项，但是没有指定域名绕过核心的选项

**Comment by eming0802** [comment]
在全局设置，访问控制，本地 IPv4 绕过地址，填入域名即可

**Reply by Vegas7** [reply]
测试了下有些域名可以生效，有些不行，不清楚为什么
